### PR TITLE
Walkthrough of AWS Services

### PR DESCRIPTION
**Pull request by ng29**
#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [x] You created a branch from `dev` branch.
- [x] You PR is raised on `dev` branch and not on`master`.
- [x] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [x] Checked for any duplicate blog entry.
- [x] Checked for all grammatical errors in the blog.
- [x] You have not published the blog content anywhere else on the internet on platforms like medium, dev.to etc.
- [x] The content is not copy-pasted from anywhere else on the Internet.

##### Author Details for Blog Post Entry
id: Nitin Gupta
bio: Software Engineer in Fintech organization and loves to code
github:ng29
linkedin:https://www.linkedin.com/in/ng2906

### CODE REVIEW

**Comment by vyasriday** [comment]
@ng29 Thanks for contributing, Can you please add some more details in other services as well as you did for EC2?

**Comment by ng29** [comment]
> @ng29 Thanks for contributing, Can you please add some more details in other services as well as you did for EC2?

Hey @vyasriday , I have mode changes as per my expertise, hope it will suffice.. 

Cheers.

**Comment by vyasriday** [comment]
@ng29 Thanks for updating it. I will review it and let you know if further changes required.

**Comment by ng29** [comment]
> @ng29 Thanks for updating it. I will review it and let you know if further changes required.

Thanks @vyasriday , hope it will get counted for Hacktoberfest.  ! !

File: content/blog/a-journey-with-AWS/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+## AWS Lex
+Amazon Lex provides advanced deep learning functionalities of ASR(Automatic Speech Recognition) & NLU(Natural Language Understanding). With the use of Lex, we can build chatbots that can converse using speech and text as mediums.
+There are a few components of BOT that need to be understood in order to build it.
+
+- Intent:- It is a particular goal that the user wants to achieve.
+- Utternaces:-These are spoken/typed phrases that invoke content.
+- Slots:- Data provided by the user to fulfill the intent.
+- Prompts:-These are queries/questions asked by the user to input the data.
+
+Moreover, there are VERSIONS mapped to bots, intents, and custom slots whenever we create we have to provide version ,main agenda of version is to make Immutable versions of bot that is created using AWS Lex.
```
Comment (vyasriday) [inline content/blog/a-journey-with-AWS/index.md:78]: This paragraph is not clear. Please rewrite it better clarity.

**Review by vyasriday — CHANGES_REQUESTED** [review]

**Comment by vyasriday** [comment]
@ng29 Please update as per review. I have made a lot of grammatical fixes so please take a pull before making changes.

**Comment by ng29** [comment]
> @ng29 Please update as per review. I have made a lot of grammatical fixes so please take a pull before making changes.

Done from my side.

File: content/blog/a-journey-with-AWS/index.md (hunk 1 above)
Comment (ng29) [reply content/blog/a-journey-with-AWS/index.md]: Done.

**Comment by ng29** [comment]
Hi @puneetsingh24  / @vyasriday  kindly review and merge pull request so that it can be counted into my HACKTOBERFEST contribution. 

Cheers

**Review by vyasriday — APPROVED** [review]

**Pull request merged**